samples/authSample.html

```diff
@@ -91,32 +94,28 @@
       }
 
       function handleAuthClick(event) {
-        auth2.signIn();
+        gapi.auth2.getAuthInstance().signIn();
       }
 
       function handleSignoutClick(event) {
-        auth2.signOut();
+        gapi.auth2.getAuthInstance().signOut();
       }
 
       // Load the API and make an API call.  Display the results on the screen.
       function makeApiCall() {
-        gapi.client.load('people', 'v1', function() {
-          var request = gapi.client.people.people.get({
-            resourceName: 'people/me'
-          });
-          request.execute(function(resp) {
-            var p = document.createElement('p');
-            var name = resp.names[0].givenName;
-            p.appendChild(document.createTextNode('Hello, '+name+'!'));
-            document.getElementById('content').appendChild(p);
-          });
+        gapi.client.people.people.get({
+          resourceName: 'people/me'
+        }).then(function(resp) {
+          var p = document.createElement('p');
+          var name = resp.result.names[0].givenName;
+          p.appendChild(document.createTextNode('Hello, '+name+'!'));
+          document.getElementById('content').appendChild(p);
         });
-        // Note: In this example, we use the People API to get the current
-        // user's name. In a real app, you would likely get basic profile info
-        // from the GoogleUser object to avoid the extra network round trip.
-        console.log(auth2.currentUser.get().getBasicProfile().getGivenName());
       }
     </script>
-    <script src="https://apis.google.com/js/api.js?onload=handleClientLoad"></script>
+    <script async defer src="https://apis.google.com/js/api.js" 
+      onload="handleClientLoad()" 
```

Comment:
this should also include this.onload = function() {}; to ensure one-shot callback